integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -848,12 +850,15 @@ public void testCreateNewCluster() {
    * To: "image: container-registry.oracle.com/middleware/weblogic:14.1.1.0-11"
    * Verify all the pods are restarted and back to ready state
    * Verify the admin server is accessible and cluster members are healthy
+   * This test will be aborted if the image tag is: 14.1.1.0-11
    */
   @Order(5)
   @Test
   @DisplayName("Verify server pods are restarted by updating image name")
   public void testUpdateImageName() {
-
+    logger.info("In the testUpdateImageName() the image version is: " + WEBLOGIC_IMAGE_TAG);
```

Comment:
`  class WebLogicImageCondition implements ExecutionCondition {

    @Override
    public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
      if (WEBLOGIC_IMAGE_TAG.equals(WLS_UPDATE_IMAGE_TAG)) {
        return ConditionEvaluationResult.
            disabled(String.format("No latest image availbale to continue test. Skipping test!"));
      } else {
        return ConditionEvaluationResult.
            enabled(String.format("Updating image to '%s'. Continuing test!", WLS_UPDATE_IMAGE_TAG));
      }
    }
  }

  @Retention(RetentionPolicy.RUNTIME)
  @ExtendWith(WebLogicImageCondition.class)
  @interface AssumeWebLogicImage {
  }`

Reply:
Then you can annotate the test method with @AssumeWebLogicImage

Reply:
@sankarpn Thanks for your input. Working on it.